### DISCUSSION TITLE
All vercel.app domains not working on my network

### DISCUSSION THREAD

**Discussion by nahasco** [vercel/community#6296] [Help]
### Summary

Titles explains it.

Any .vercel.app domain in all of my projects is simply not working and the browser is throwing a 

```
This site can’t be reached url took too long to respond.
Try:

Checking the connection
[Checking the proxy and the firewall](chrome-error://chromewebdata/#buttons)
ERR_TIMED_OUT
```

Pagespeed can test the website so its working but its not for my network, tried multiple devices and browsers but no luck.

Whats going on? How to fix this and why did it happen?

### Steps to Reproduce

Cant be reproduced since its a local issue

**Comment by amyegan** [comment]
Are you able to access the sites from a different network?

**Comment by usifosuo** [comment]
I am having the same issue my Vercel.app not working on other networks or devices. What is causing this problem and how can I fix please?

**Reply by Codacad** [reply]
I got it right now, create a new deployment, it will work.

I created a deployment, it worked.

**Comment by Codacad** [comment]
I am also having this issue. Is there any solution for this please?

**Comment by Codacad** [comment]
I got it right now, create a new deployment, it will work.

I created a deployment, it worked.
